Makefile

```diff
@@ -261,6 +261,14 @@ clean.go: ; $(info $(H) cleaning...)
 	$(eval GO_CLEAN_FLAGS := -i -r)
 	$(Q) $(GO) clean $(GO_CLEAN_FLAGS)
 
+#-----------------------------------------------------------------------------
+# Target: unit-test
+#-----------------------------------------------------------------------------
+.PHONY: unit-test
+
+unit-test:
+	GOOS=linux CGO_ENABLED=1 go test -v ./cmd/istio-cni/...
```

Comment:
shouldn't have to set GOOS here.  the tests should build with whatever arch it's on.